west.yml

```diff
@@ -141,7 +141,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: 741729d16c8ab0dd537d802d480180786951acad
+      revision: pull/1081/head
```

Comment:
Nothing to do with the line but fix your commit message.

Reply:
Thanks, fixed.